lang/en.json

```diff
@@ -388,5 +388,10 @@
   "YZECORIOLIS.NewShipProblem": "New Problem",
   "YZECORIOLIS.ShipEditProblem": "Edit Problem",
   "YZECORIOLIS.ShipCreateProblem": "Create Problem",
-  "YZECORIOLIS.ShipDeleteProblem": "Delete Problem"
+  "YZECORIOLIS.ShipDeleteProblem": "Delete Problem",
+
+  "YZECORIOLIS.ItemDeleteConfirmationTitle": "Delete Item",
+  "YZECORIOLIS.ItemDeleteConfirmationContent": "Are you sure you want to delete this item?",
+  "YZECORIOLIS.Yes": "Yes",
```

Comment:
Not sure if these are already available in core Foundry, if so, I couldn't find it.